client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.controller.js

```diff
@@ -46,6 +47,13 @@ angular.module("managerApp").controller("TelecomTelephonyServiceContactCtrl", fu
         }).catch(function (err) {
             return new ToastError(err);
         }).finally(function () {
+            OvhApiTelephony.Lexi().schema({}, {}).$promise.then(function (infos) {
```

Comment:
Use a ```$q.all``` and return it in the init method. For the time this request take, you can block bulk during the whole initialization.